packages/terra-progress-bar/src/ProgressBar.jsx

```diff
@@ -27,19 +27,25 @@ const propTypes = {
    * Boolean check for gradient.
    */
   hasGradient: PropTypes.bool,
+  /**
+   * Value passed to aria-valuetext for accessibility
+   */
+  valueText: PropTypes.string,
```

Comment:
t seems like this prop should be required if needed for accessibly. Because we need to be passive, should we
1) log an issue to make this prop required in the next MVB
2) and add a non-prod warning message saying this prop will be made required in the next MVB and should be supplied for accessibility?

Reply:
I don't think this attribute is always required, reading through the documentation is sounds like screen readers should be able to interpret the valueNow as a percentage and the valueText is usually used to represent non-numeric / percentage based valueNows. But it does make me wonder if we should be locking down the other aria attributes for min, max, and valueNow

https://www.w3.org/WAI/PF/aria/states_and_properties#aria-valuetext

https://www.w3.org/WAI/PF/aria/states_and_properties#aria-valuenow

Reply:
What are you thinking of locking them down to?

Reply:
If this shouldn't be required, can we beef up the description to describe when this prop should be used with a link to the accessibility information? I imagine consumers would find that useful!